examples/slackbot/Notebook.ipynb

```diff
@@ -74,120 +67,89 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "e95d4d50",
-   "metadata": {
-    "tags": []
-   },
+   "id": "7e6fedb9",
+   "metadata": {},
    "outputs": [],
    "source": [
-    "SYSTEM_CONTEXT = \"\"\"\n",
-    "You are a helpful assistant designed to suggest the Slack usernames of \n",
-    "people who need to know about a Slack conversation.\n",
-    "\n",
-    "Only respond as a JSON list containing the Slack usernames of people to notify \n",
-    "of the conversation, or return an empty list if no should be notified.\n",
+    "def build_prompts(messages):\n",
+    "    message_time = messages.col(\"ts\")\n",
+    "    last_message_time = message_time.lag(1) # !!!\n",
+    "    is_new_conversation = message_time - last_message_time > 10 * 60\n",
```

Comment:
We could get `seconds_between`. Getting a cleaner "single timedelta" approach will require more work -- Arrow has different kinds of durations for things like "Calendar Days", so a single "timedelta" may actually be multiple things (when you say days do you mean with or without leap seconds, etc.)

Reply:
OK - going to call this `seconds_since` so the base operand is more clear.